docs/overview.md

```diff
@@ -44,10 +44,12 @@ In comparison to Ethereum, Gno.land offers distinct advantages:
    usability beyond the context of blockchain. In contrast, Solidity is designed specifically for Smart Contracts on the
    Ethereum platform.
 
-## Gno.land Documentation Overview
+## Using the Gno.land Documentation
 
 Gno.land's documentation adopts the [Diataxis](https://diataxis.fr/) framework, ensuring structured and predictable content. It includes:
+- A [Getting Started](./getting-started/local-setup.md) section, covering simple instructions on how to begin your journey into Gno.land.
+- Concise how-to guides for specific technical tasks.
```

Comment:
```suggestion
- Concise how-to-guides for specific technical tasks.
```

Reply:
This is actually already correct grammar-wise.